### PR TITLE
added hooks to announcement class

### PR DESCRIPTION
**Pull request by nasir9bd**

### CODE REVIEW

File: modules/hrm/includes/class-announcement.php [hunk 1]
```diff
@@ -196,6 +196,9 @@ function meta_boxes_cb( $post_id ) {
                         </select>
                     </td>
                 </tr>
+
+                <?php do_action( 'hr-announcement-sms-settings', $post ); ?>
```
Comment (tareq1988) [inline modules/hrm/includes/class-announcement.php]: You can not add `sms` in the action name. Use something generic, like `hr_accouncement_table_last`

File: modules/hrm/includes/class-announcement.php [hunk 2]
```diff
@@ -309,6 +312,8 @@ function save_announcement_meta( $post_id, $post ) {
         update_post_meta( $post_id, '_announcement_type', $announcement_assign_type );
         update_post_meta( $post_id, '_announcement_selected_user', $announcement_assign_employee );
 
+        do_action( 'hr-annoucement-sms-settigs-save', $post_id );
```
Comment (tareq1988) [inline modules/hrm/includes/class-announcement.php]: `hr_announcement_save`

Comment (nasir9bd) [inline modules/hrm/includes/class-announcement.php]: ok

File: modules/hrm/includes/class-announcement.php [hunk 3]
```diff
@@ -420,6 +425,8 @@ function insert_assign_employee( $employee_array, $post_id ) {
 
         $sql = "INSERT INTO {$table_name} (`user_id`, `post_id`, `status` ) VALUES $values";
         $wpdb->query( $sql );
+
+        do_action( 'hr-announcement-send-sms', $employee_array, $post_id );
```
Comment (tareq1988) [inline modules/hrm/includes/class-announcement.php]: `hr_announcement_insert_assignment`

**Pull request closed without merging**